File: core/src/test/java/com/toocol/ssh/core/mosh/core/network/ConnectionTest.java
package com.toocol.ssh.core.mosh.core.network;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

/**
 * @author ：JoeZane (joezane.cn@gmail.com)
 * @date: 2022/6/29 0:55
 * @version:
 */
class ConnectionTest {

    @Test
    void timeout() {
        Transport.Addr addr = new Transport.Addr("localhost", 60001, "WkQ1ElbK11SJywYUggbl7g");
        Connection connection = new Connection(addr, null);
        assertEquals(1000, connection.timeout());
    }

}

File: core/src/test/java/com/toocol/termio/core/mosh/core/network/ConnectionTest.kt
package com.toocol.termio.core.mosh.core.network

import com.toocol.termio.core.mosh.core.network.Transport.Addr
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

/**
 * @author ：JoeZane (joezane.cn@gmail.com)
 * @date: 2022/6/29 0:55
 * @version:
 */
internal class ConnectionTest {
    @Test
    fun timeout() {
        val addr = Addr("localhost", 60001, "WkQ1ElbK11SJywYUggbl7g")
        val connection = Connection(addr, null)
        Assertions.assertEquals(1000, connection.timeout())
    }
}